### DISCUSSION TITLE
How to build the latest release in Visual Studio?

### DISCUSSION THREAD

**Discussion by MenonPlayz** [34736384/genshin-fps-unlock#331] [Q&A] [answered]
I downloaded the source code, opened the .sln solution file in Visual Studio 2022 and clicked build. It produces a .exe along with some other files (seen in the picture), but when I move the exe to another directory it doesn't work anymore.
![grafik](https://github.com/34736384/genshin-fps-unlock/assets/90968284/bdd21b5b-9589-4adf-ab25-d3fbcabdfa7e)
I found some threads on the internet saying to select publish and produce a self-contained exe, but that exe is 150MB big and also doesnt work outside of the build directory.

If someone knows how to properly build the source to turn out a single exe like we can download, please give us a short guide

**Comment by 34736384** [comment, marked as answer]
use framework dependent option instead

**Reply by MenonPlayz** [reply]
![grafik](https://github.com/34736384/genshin-fps-unlock/assets/90968284/e1216576-d687-4643-ba33-8e95843a9ad3)

Like this?

**Reply by 34736384** [reply]
yes

**Reply by MenonPlayz** [reply]
Thanks, it works! I hope this helps other people as well.
Still gets detected and quarantined by defender though lol (Trojan:Win32/Wacatac.B!ml)
